lib/utils/load.rb

```diff
@@ -165,9 +165,14 @@ def load_env_and_vars(structure)
   load_vars(structure)
 end
 
+def load_env_and_vars_no_commands(structure)
```

Comment:
It's cleaner to just add a new optional flag parameter to `load_env_and_vars` rather than make a separate, almost identical method

Reply:
yeah, just merged it into one